docs/index.md

```diff
@@ -6,9 +6,49 @@ we have a library of research experiments using Cirq -- [ReCirq](https://github.
 This repository contains code for our flagship experiments, enabling
 you to reproduce and extend cutting edge quantum computing research.
 
- - QAOA, [arxiv](https://arxiv.org/abs/2004.04197).
- - Hartree-Fock VQE, [Science (2020)](https://science.sciencemag.org/content/369/6507/1084) | [arxiv](https://arxiv.org/abs/2004.04174)
- - Fermi-Hubbard, [arxiv](https://arxiv.org/abs/2010.07965).
- - Quantum Chess, [arxiv](https://arxiv.org/abs/1906.05836).
+## Audience
```

Comment:
```suggestion
## Quantum researchers
```

Good to be (overly) direct in the headings since 1. users tend to skim, and 2. search engines seem to like headings/titles